src/cfml/system/services/ServerEngineService.cfc

```diff
@@ -50,20 +50,28 @@ component accessors="true" singleton="true" {
 	* @version Version number or empty to use default
 	**/
 	public function installAdobe( required destination, required version ) {
-		var installDetails = installEngineArchive( 'adobe@#version#', destination );			
+		var installDetails = installEngineArchive( 'adobe@#version#', destination );	
+
 		// set password to "commandbox"
 		// TODO: Just make this changes directly in the WAR files
 		fileWrite( installDetails.installDir & "/WEB-INF/cfusion/lib/password.properties", "rdspassword=#cr#password=commandbox#cr#encrypted=false" );
 		// set flex log dir to prevent WEB-INF/cfform being created in project dir
 		if (fileExists(installDetails.installDir & "/WEB-INF/cfform/flex-config.xml")) {
 			var flexConfig = fileRead(installDetails.installDir & "/WEB-INF/cfform/flex-config.xml");
-			// escape back slashes,forward slashes, periods and colons in the installDir for the regular expression
-			var escapedInstDir=reReplace(installDetails.installDir,"(\\|\/|\.|\:)","\\1","all");
-			//ugh. reReplace supports 5 backreferences in the replacement substring for case conversions. It doesn't allow escaping of these backreferences though in case you want them literally! 
-			var replaceString=reReplace(installDetails.installDir,"\\(u|U|l|L|E)","##chr(92)##\1","all");
-			var regex="(?:" & escapedInstDir & ")*" & "\/WEB-INF\/" ;
-			flexConfig = reReplace(flexConfig, regex, replaceString & "/WEB-INF/","all");
-			fileWrite(installDetails.installDir & "/WEB-INF/cfform/flex-config.xml", evaluate(de(flexConfig)));
+
+			if(!installDetails.internal && installDetails.initialInstall ) {
```

Comment:
FWIW, `installDetails.internal` will never be true for any Adobe server ATM since the internal flag is when you start the embedded Lucee jars that are part of the CLI and don't donwload a new WAR.  The code isn't hurting anything-- just thought I'd explain that.